packages/@sanity/form-builder/src/inputs/files/common/ImageRatio.tsx

```diff
@@ -0,0 +1,15 @@
+import {Box} from '@sanity/ui'
+import styled from 'styled-components'
+
+export const ImageRatio = styled(Box)<{ratio?: number}>`
```

Comment:
Maybe call this `RatioBox`, since it’s could be used for non-image scenarios?

Reply:
Good idea! Fixed.